pkgs/development/python-modules/onnxconverter-common/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, protobuf, numpy, requests, onnx }:
+
+buildPythonPackage rec {
+  pname = "onnxconverter_common";
+  version = "1.7.0";
+
+  propagatedBuildInputs = [ protobuf numpy requests onnx ];
+  format = "wheel";
+  src = fetchPypi {
+    inherit pname version;
+    format = "wheel";
```

Comment:
It's preferable to not use wheels, do you mind fetching from github: https://github.com/microsoft/onnxconverter-common

Reply:
@jonringer It looks like pypi only provides a wheel: https://pypi.org/project/onnxconverter-common/1.7.0/#files
So are you suggesting that I `fetchFromGitHub` and build from source, rather than using `fetchPypi`?

Reply:
> @jonringer It looks like pypi only provides a wheel: [pypi.org/project/onnxconverter-common/1.7.0/#files](https://pypi.org/project/onnxconverter-common/1.7.0/#files)
> So are you suggesting that I `fetchFromGitHub` and build from source, rather than using `fetchPypi`?

Yes

Reply:
Please fetch from github.